crates/orchestrator/src/cli/database/mongodb.rs

```diff
@@ -0,0 +1,18 @@
+use clap::Args;
+
+/// Parameters used to config MongoDB.
+#[derive(Debug, Clone, Args)]
+#[group(requires_all = ["mongodb_connection_url"])]
```

Comment:
why not database name?

Reply:
Even tho the user can provide it if they wish, I wanted to keep it default to `orchestrator` whenever possible